### PR TITLE
Using gulp to build standalone version of 'mixins'

### PR DESCRIPTION
**Pull request by Johnius**
First simple version. Any comments?

### CODE REVIEW

**Comment by Yomguithereal** [comment]
Hello @Johnius. Thanks for your PR. One remark:

Why use gulp here since the repository doesn't use it yet and since the build could be done without it in a package.json script?

File: gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+var gulp = require('gulp');
+var gutil = require('gulp-util');
+var source = require('vinyl-source-stream');
+var browserify = require('browserify');
+
+gulp.task("build:mixins", function() {
+  var b = browserify({
+    entries: './build.js'
```
Comment (Yomguithereal) [inline gulpfile.js]: What is the `./build.js` file?

Comment (Yomguithereal) [inline gulpfile.js]: Nevermind. I found it. Sorry.

**Comment by Yomguithereal** [comment]
Something like:

```json
{
  "scripts": {
    "build": "npm build-mixins",
    "build-mixins": "mkdir build && browserify ./src/mixins.js -o ./build/mixin.js"
  }
}
```

And then we'll add the other patterns?

**Comment by Johnius** [comment]
I didn't thought about it in this way. I used what I knew.

I'll make it in the way you said.

**Comment by Yomguithereal** [comment]
Hello @Johnius, I've added the command to `package.json`. Following discussion on #40.

**Pull request closed without merging**